metabase/metabase
# Make engine widget accessible

Epic: https://github.com/metabase/metabase/issues/19696

How to test:
- Run a new instance
- Go to database setup step
- Make sure the input is autofocused
- Navigate between databases via arrows
- Select a database with `Enter`
- The button to remove the databaes should be autofocused
- Remove the database with `Enter`
